website/docs/reference/resource-configs/database.md

```diff
@@ -5,10 +5,47 @@ datatype: string
 description: "Read this guide to understand the database configuration in dbt."
 ---
 
-:::caution Heads up!
-This is a work in progress document. While this configuration applies to multiple resource types, the documentation has only been written for seeds.
+<Tabs>
+<TabItem value="model" label="Model">
```

Comment:
Is it possible to put tests into a custom database? I assume that's alongside store-failures behaviour? Inferring this from the resource_types array above.

Reply:
hmm good question! have askd in dbt help to confirm but i also did add an example just incase